src/main/java/hm25/HeroConfiguration.java

```diff
@@ -0,0 +1,56 @@
+package hm25;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import hm21.HeroDao;
+import hm21.HeroDaoImpl;
+import hm22.HeroFabric;
+import hm22.HeroMovieService;
+import hm22.HeroService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ComponentScan("java")
```

Comment:
`package hm25; `
пакет невірний. 

```suggestion
@ComponentScan("hm25")
```